### PR TITLE
Add assemblycomparator2

### PR DESCRIPTION
**Pull request by cmkobel**
add assemblycomparator2 with new binary.

https://github.com/cmkobel/assemblycomparator2

assemblycomparator2 (asscom2) is a genomes-to-report pipeline. It is a bit like nullarbor, but it takes in genomes (assemblies) instead of reads. Assemblies can come from isolates or metagenomes - as long as they're all prokaryotic.

### CODE REVIEW

**Comment by cmkobel** [comment]
@BiocondaBot please add label

**Comment by cmkobel** [comment]
Hello
Please let me know if I need to do something in order to get this into bioconda.
Best, Carl

File: recipes/assemblycomparator2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package:
+  name: assemblycomparator2
+  version: "2.5.6"
+
+source:
+  #url: https://github.com/cmkobel/assemblycomparator2/archive/refs/tags/v2.5.5.tar.gz [linux]
+  #md5: 38c288550d05d44e1c5c6682ecccdc23 [linux]
+  #url: https://github.com/cmkobel/assemblycomparator2/archive/refs/heads/master.zip # [linux]
+  #md5: 5c84ad46eb1c385a07332e6f9945166a # [linux]
+  url: https://github.com/cmkobel/assemblycomparator2/archive/refs/tags/v2.5.6.tar.gz
```
Comment (thanhleviet) [inline recipes/assemblycomparator2/meta.yaml:10]: Please use variable for version, name like this example https://github.com/bioconda/bioconda-recipes/blob/master/recipes/hisat2/meta.yaml#L8

File: recipes/assemblycomparator2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#  skip: True [not osx]
+
+# Maybe I should tryy 
+# skip: True  #[osx]
+
+
+requirements:
+  run:
+    - snakemake >=7.32.3
+    - mamba >=1.4.9
```
Comment (thanhleviet) [inline recipes/assemblycomparator2/meta.yaml:33]: Are there any more bioinformatics dependencies for **assemblycomparator2**, why did you want to include mamba?

File: recipes/assemblycomparator2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  #md5: 38c288550d05d44e1c5c6682ecccdc23 [linux]
+  #url: https://github.com/cmkobel/assemblycomparator2/archive/refs/heads/master.zip # [linux]
+  #md5: 5c84ad46eb1c385a07332e6f9945166a # [linux]
+  url: https://github.com/cmkobel/assemblycomparator2/archive/refs/tags/v2.5.6.tar.gz
+  md5: 2ffd351078346dce8de117f973df5efc
+
+
+build:
+  number: 0
+  run_exports: snakemake
```
Comment (thanhleviet) [inline recipes/assemblycomparator2/meta.yaml:16]: Please consult the docs of using run_export at

https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#export-runtime-requirements

https://bioconda.github.io/contributor/linting.html#missing-run-exports

Essentially it's assemblycomparator2, not snakemake

File: recipes/assemblycomparator2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# This is correct but fails linting
+#build:
+#  number: 0
+#  run_exports: snakemake
+#  skip: True [not osx]
+
+# Maybe I should tryy 
+# skip: True  #[osx]
+
+
```
Comment (thanhleviet) [inline recipes/assemblycomparator2/meta.yaml:29]: Do you want to add the test section https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#test-section?

File: recipes/assemblycomparator2/run_test.sh [hunk 5]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```
Comment (thanhleviet) [inline recipes/assemblycomparator2/run_test.sh:1]: How is this script used in this recipe?

**Comment by cmkobel** [comment]
@BiocondaBot rebuild

**Review by thanhleviet — APPROVED** [review]
LGTM

**Comment by cmkobel** [comment]
Hi Thanh 
Thank you very much for reviewing my recipe. I see that I have misunderstood a few things but I think I was able to figure most things out thanks to your directions. So thank you very much.
The test is still failing because of some arbitrary temporary-directory writing permissions that I have to fix. 
Is it commonplace to add the reviewer to the recipe-maintainers list?

**Comment by cmkobel** [comment]
@thanhleviet Ready to merge.

**Pull request merged**